orangecontrib/spectroscopy/widgets/preprocessors/misc.py

```diff
@@ -295,34 +295,40 @@ def createinstance(cls, params):
         return SavitzkyGolayFiltering(window=window, polyorder=polyorder, deriv=deriv)
 
 
-class CurveShiftEditor(BaseEditorOrange):
+class LinearTransformEditor(BaseEditorOrange):
     """
-    Editor for CurveShift
+    Editor for LinearTransform
     """
     # TODO: the layout changes when I click the area of the preprocessor
     #       EFFECT: the sidebar snaps in
 
-    name = "Shift Spectra"
-    qualname = "orangecontrib.infrared.curveshift"
+    name = "Linear Transformation"
+    qualname = "orangecontrib.infrared.lineartransform"
```

Comment:
Since we are renaming anyway, this is a good time to drop the legacy `infrared` namespace: use `orangecontrib.spectroscopy.lineartransform` instead.

Reply:
okay, i think i resolved these changes in the next 2 commits